clients/java/src/test/java/io/camunda/client/OAuthCredentialsProviderTest.java

```diff
@@ -460,6 +483,54 @@ private void mockCredentials(final String token, final String scope) {
     }
   }
 
+  private void mockCredentialsWithResource(
```

Comment:
Could you please let `mockCredentials(final String token, final String scope)` call this method here, similar to how you did it with the tests in the zeebe package? That reduces code duplication.